packages/nx/src/daemon/client/client.ts

```diff
@@ -191,9 +191,18 @@ export class DaemonClient {
   }
 
   private async sendToDaemonViaQueue(messageToDaemon: Message): Promise<any> {
-    return this.queue.sendToQueue(() =>
-      this.sendMessageToDaemon(messageToDaemon)
-    );
+    return this.queue
+      .sendToQueue(() => this.sendMessageToDaemon(messageToDaemon))
+      .then(() => {
```

Comment:
You need to return the value resolved here right?

Reply:
Yea you're right. I finally found the reason why it was "working" and thats because the daemon got disabled so it never actually got here 🙄